### PR TITLE
Expand the README so that it becomes usable

### PR DESCRIPTION
**Pull request by lokal-profil**
Task: [T168018](https://phabricator.wikimedia.org/T168018)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,102 @@
 ## WFD_import
-A batch import of European water data based on the Water Framework Directive
-reporting.
+*WFD import* are a collection of scripts and tools for batch importing of
```
Comment (Vesihiisi) [inline README.md]: _are_ → _is_

File: README.md [hunk 2]
```diff
@@ -1,17 +1,102 @@
...
+European water data, reported under the Water Framework Directive, to Wikidata.
 
-To run it you will have to install `wikidata-stuff` using:
-`pip install git+https://github.com/lokal-profil/wikidata-stuff.git`
+It currently supports harvesting data on River Basin Districts (RBDs) and
```
Comment (Vesihiisi) [inline README.md]: Link the terms to their Wikidata items?

File: README.md [hunk 3]
```diff
@@ -1,17 +1,102 @@
...
+Surface Water Bodies (SWBs).
 
-*Note*: You might have to add the `--process-dependency-links` flag to the above
-command if you are running a different version of pywikibot from the required one.
+### How to run
+To do a basic run simply call either `RBD.py` or `swb_import.py` from the
+commandline together with the `-in_file:<path>` argument. The `-in_file` <path>
+value should be the url to an RBDSUCA .xml file for a RBD import or an SWB
```
Comment (Vesihiisi) [inline README.md]: _a RBD_ → _an RBD_

File: README.md [hunk 4]
```diff
@@ -1,17 +1,102 @@
...
 
-To run it you will have to install `wikidata-stuff` using:
-`pip install git+https://github.com/lokal-profil/wikidata-stuff.git`
+It currently supports harvesting data on River Basin Districts (RBDs) and
+Surface Water Bodies (SWBs).
 
-*Note*: You might have to add the `--process-dependency-links` flag to the above
-command if you are running a different version of pywikibot from the required one.
+### How to run
+To do a basic run simply call either `RBD.py` or `swb_import.py` from the
```
Comment (Vesihiisi) [inline README.md]: rm _simply_, it doesn't mean anything

File: README.md [hunk 5]
```diff
@@ -1,17 +1,102 @@
...
+commandline together with the `-in_file:<path>` argument. The `-in_file` <path>
+value should be the url to an RBDSUCA .xml file for a RBD import or an SWB
+file(there is one per RBD) for an SWB import.
```
Comment (Vesihiisi) [inline README.md]: Space after _file_

File: README.md [hunk 6]
```diff
@@ -1,17 +1,102 @@
...
 
-### Note
-This repo was split of from 
-[lokal-profil/wikidata_batches](https://github.com/lokal-profil/wikidata_batches)
-so the history might be a bit mixed up.
+The xml files only include the names of the RBD/SWB in English (if any), to
```
Comment (Vesihiisi) [inline README.md]: comma splice

File: README.md [hunk 7]
```diff
@@ -1,17 +1,102 @@
...
+load the names in a local language use the `-gml_file:<path>` argument together
+with the path to a .gml file belonging to the RBD or SWB dataset.
+
+You can also use the `WfdBot.load_data()` to create a json dump of either of
```
Comment (Vesihiisi) [inline README.md]: rm _the_

File: README.md [hunk 8]
```diff
@@ -1,17 +1,102 @@
...
+the .xml or .gml files. The local json dumps can then be used together with the
+`-in_file` or `gml_file` argument.
+
+To create new items, rather than just enriching pre-existing ones, use the
+`-new` flag. A new item is one where the unique RBD/SWB code is not yet
+associated with an item on Wikidata.
+
+To ensure no data is written to Wikidata you may use the `-simulate` flag. The
```
Comment (Vesihiisi) [inline README.md]: Add a sentence to explain what actually happens on the user side -- output to screen?

File: README.md [hunk 9]
```diff
@@ -1,17 +1,102 @@
...
+`-preview_file:<path>` also prevents data from being written to Wikidata,
```
Comment (Vesihiisi) [inline README.md]: General formatting note: one paragraph per flag, and possibly put the actual flag at the beginning of the line?

File: README.md [hunk 10]
```diff
@@ -1,17 +1,102 @@
...
+instead outputting a preview of the data, in wikitext, to the file at the
```
Comment (Vesihiisi) [inline README.md]: Add a link to an example preview page to exemplify what it looks like.

File: README.md [hunk 11]
```diff
@@ -1,17 +1,102 @@
...
+(CDR) under [*European Union (EU) obligations/Water Framework Directive: River Basin Management Plans - 2016 Reporting*](https://cdr.eionet.europa.eu/se/eu/wfd2016/).
+
+* The [RBDSUCA file](https://cdr.eionet.europa.eu/se/eu/wfd2016/rbdsuca/envwnulfg/RBDSUCA_SE_20170329.xml/manage_document)
+  (for Competent Authorities and RBDs) is found under *National RBDSUCA*.
+* The [SWB files](https://cdr.eionet.europa.eu/se/eu/wfd2016/districts/se1/envwntvaw/SWB_SE_20170411.xml/manage_document)
+  can be found in their respective RBD directories under *River Basin Districts*.
+* Both .gml files (one for RBDs and one for SWBs) are located under
+  [*National spatial data*](https://cdr.eionet.europa.eu/se/eu/wfd2016/spatial/envwdqi7a/).
+
+It is worth nothing that there may be multiple releases of data (separate
```
Comment (Vesihiisi) [inline README.md]: _nothing_ → _noting_

File: README.md [hunk 12]
```diff
@@ -1,17 +1,102 @@
...
+directories), always chose the most recent. For the .gml files it varies from
```
Comment (Vesihiisi) [inline README.md]: Comma splice. How about _so you should always choose the most recent one_...

File: README.md [hunk 13]
```diff
@@ -1,17 +1,102 @@
...
+country to country whether either of them are made accessible to the public.
+
+### How to add a new country
+Before investing energy into adding support for a new country you should ensure
+that the license of the data is compatible with Wikidata (CC0). The default
```
Comment (Vesihiisi) [inline README.md]: Link the license names to their respective deeds?

File: README.md [hunk 14]
```diff
@@ -1,17 +1,102 @@
...
+
+During the run the above additions are validated to ensure all the required
+mappings are present. Feel free to add any new mappings as a pull request to
+this repository.
+
+To import the data first run the RBD importer (with `-new` if needed) before
+running the SWB importer.
+
+### Known issues and limitations
+The results are of curse limited by the data quality. If a country has not
```
Comment (Vesihiisi) [inline README.md]: _curse_ → _course_

File: README.md [hunk 15]
```diff
@@ -1,17 +1,102 @@
...
+
+RBD does not make use of `internationalRBDName` (nor does either SWB or RBD make
+use of `nameTextInternational`). Although this is supposed to be an international
+(English) label the field was found to hold a variety of content.
 
 ### Issues
 Issues and bugs are tracked on [Phabricator](https://phabricator.wikimedia.org/tag/wmse-wfd-data-to-wikidata-2016/).
+
+### Installation
+if `pip -r requirements.txt` does not work correctly you might have to add
```
Comment (Vesihiisi) [inline README.md]: Start of sentence, capital letter.

File: README.md (hunk 1 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (lokal-profil) [inline README.md]: Not a bad idea =). Done

File: README.md (hunk 4 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 3 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 5 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 7 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 8 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 9 above)
Comment (lokal-profil) [inline README.md]: Changed to one per paragraph for each. Sticking them at the start would however require a substantial restructuring. For users who want it that way they can just run the script with the `-help` flag. (added a not about all common pywikibot flags being supported).

File: README.md (hunk 10 above)
Comment (lokal-profil) [inline README.md]: Done

File: README.md (hunk 11 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 13 above)
Comment (lokal-profil) [inline README.md]: Done

File: README.md (hunk 14 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 15 above)
Comment (lokal-profil) [inline README.md]: Done

File: README.md (hunk 6 above)
Comment (lokal-profil) [inline README.md]: done

File: README.md (hunk 12 above)
Comment (lokal-profil) [inline README.md]: Done and rephrased a bit

**Pull request merged**